src/audio/tdfb/tdfb_math.c

```diff
@@ -0,0 +1,204 @@
+/* Temporary module that will be removed */
```

Comment:
This will be in @ShriramShastry 's PR.

Reply:
It would be best if DRC would remove the internal math functions and use math lib once it's ready for use.